qa/rspec/commands.rb

```diff
@@ -21,7 +21,6 @@
 require_relative "./commands/suse"
 require_relative "./commands/centos/centos-6"
 require_relative "./commands/oel/oel-6"
-require_relative "./commands/ubuntu/ubuntu-1604"
```

Comment:
Ubuntu 1604 is still in the `qa/config/platforms.json`, so it is unclear why we are removing specifics for it in this PR. Since we no longer require the `qa/rspec/commands/ubuntu/ubuntu-1604` file, I believe if this change is intentional then we can also delete the orphaned file.

Reply:
It can be removed. Once the base ubuntu version was updated, all the Ubuntu distributions we test use  `::ServiceTester::SystemD`, removing the need for the specific subclass.